saleor/graphql/page/mutations/pages.py

```diff
@@ -123,6 +128,11 @@ class Meta:
         error_type_class = PageError
         error_type_field = "page_errors"
 
+    @classmethod
+    def save(cls, info, instance, cleaned_input):
+        super(PageCreate, cls).save(info, instance, cleaned_input)
```

Comment:
Should we run `info.context.plugins.page_created(instance)` when `Page` is updated?

Reply:
As you call `save` method from `PageCreate`, the `page_created` method is called too.

Reply:
What about that?